yt_dlp/extractor/bbc.py

```diff
@@ -631,14 +633,14 @@ class BBCIE(BBCCoUkIE):  # XXX: Do not subclass from concrete IE
         'info_dict': {
             'id': 'p02mprgb',
             'ext': 'mp4',
-            'title': 'Aerial footage showed the site of the crash in the Alps - courtesy BFM TV',
-            'description': 'md5:2868290467291b37feda7863f7a83f54',
-            'duration': 47,
+            'title': 'Germanwings crash site aerial video',
+            'description': r're:(?s)Aerial video showed the site where the Germanwings flight 4U 9525, .{156} BFM TV\.$',
+            'duration': None,  # 47,
```

Comment:
> The rationale for commenting out a known bad test value is to avoid that error masking a problem with the expected good values, since the test harness only finds one bad value at a time.

Devs should do that locally, not push it into PR